Cargo.toml

```diff
@@ -52,41 +52,44 @@ doomsday = ["task", "sync"]
 http-client = [
     "rust_tls",
     "async-std/default",
-    "dep:tokio",
-    "dep:hyper",
-    "dep:anyhow",
-    "dep:webpki-roots",
+    "dep:bytes",
     "dep:http",
+    "dep:hyper",
     "dep:serde",
     "dep:serde_json",
+    "dep:tokio",
     "dep:once_cell",
+    "dep:webpki-roots",
 ]
 tokio1 = ["async-std/tokio1"]
 http-client-json = ["http-client"]
 # useful for tests
 __skip-http-client-cert-verification = []
 
 [dependencies]
+anyhow = { version = "1.0" }
 log = "0.4.0"
+thiserror = "1.0"
```

Comment:
A minor nit, can we list by alphabetically so it's easier to find

Reply:
Yup, I did a partial earler and was debating if I should redo it further... updated